sample/src/main/java/com/schibsted/spain/barista/sample/widget/SampleImageView.kt

```diff
@@ -0,0 +1,13 @@
+package com.schibsted.spain.barista.sample.widget
+
+import android.content.Context
+import android.util.AttributeSet
+import androidx.appcompat.widget.AppCompatImageView
+
+class SampleImageView @JvmOverloads constructor(
```

Comment:
As this class is only intended to exist because we need it to run tests on it I would suggest adding a comment to explain it. Right now it seems a good candidate to be deleted.

Reply:
I've deleted it , as in fact it wasn't doing much, and will be on the library code... not needed